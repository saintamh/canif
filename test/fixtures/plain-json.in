[
{
"name": "Suspended Routes",
"id": 1274430,
"displayOnLoad": false
},
{
"name": "Roosevelt Island Operating Corporation",
"id": 1274431,
"displayOnLoad": true
},
{
"name": "Huntington Area Rapid Transit",
"id": 1274432,
"displayOnLoad": true
},
{
"name": "Ferries",
"id": 1274433,
"displayOnLoad": true
},
{
"name": "Putnam Transit",
"id": 1274434,
"displayOnLoad": true
},
{
"name": "Suffolk County Transit",
"id": 1274435,
"displayOnLoad": true
},
{
"name": "Nassau Inter-County Express",
"id": 1274436,
"displayOnLoad": true
},
{
"name": "Bee-Line Bus System",
"id": 1274437,
"displayOnLoad": true
},
{
"name": "NJ Transit Bus",
"id": 1274438,
"displayOnLoad": true
},
{
"name": "New York City Bus",
"id": 1274441,
"displayOnLoad": true
},
{
"name": "New York City Bus",
"id": 1274439,
"displayOnLoad": true
},
{
"name": "AirTrain Newark",
"id": 1274440,
"displayOnLoad": true
},
{
"name": "AirTrain JFK",
"id": 1274442,
"displayOnLoad": true
},
{
"name": "AirTrain JFK",
"id": 1274443,
"displayOnLoad": true
},
{
"name": "NJ Transit Rail",
"id": 1274444,
"displayOnLoad": true
},
{
"name": "NJ Transit Rail",
"id": 1274445,
"displayOnLoad": true
},
{
"name": "Metro-North Railroad",
"id": 1274446,
"displayOnLoad": true
},
{
"name": "Metro-North Railroad",
"id": 1274447,
"displayOnLoad": true
},
{
"name": "Long Island Rail Road",
"id": 1274449,
"displayOnLoad": true
},
{
"name": "Long Island Rail Road",
"id": 1274448,
"displayOnLoad": true
},
{
"name": "PATH",
"id": 1274450,
"displayOnLoad": true
},
{
"name": "PATH",
"id": 1274451,
"displayOnLoad": true
},
{
"name": "Staten Island Railway",
"id": 1274452,
"displayOnLoad": true
},
{
"name": "Staten Island Railway",
"id": 1274453,
"displayOnLoad": true
},
{
"name": "New York City Subway",
"id": 1274454,
"displayOnLoad": true
},
{
"name": "New York City Subway",
"id": 1274455,
"displayOnLoad": true
}
]
