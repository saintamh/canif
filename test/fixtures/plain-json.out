[
    {"displayOnLoad": false, "id": 1274430, "name": "Suspended Routes"},
    {
        "displayOnLoad": true,
        "id": 1274431,
        "name": "Roosevelt Island Operating Corporation"
    },
    {
        "displayOnLoad": true,
        "id": 1274432,
        "name": "Huntington Area Rapid Transit"
    },
    {"displayOnLoad": true, "id": 1274433, "name": "Ferries"},
    {"displayOnLoad": true, "id": 1274434, "name": "Putnam Transit"},
    {
        "displayOnLoad": true,
        "id": 1274435,
        "name": "Suffolk County Transit"
    },
    {
        "displayOnLoad": true,
        "id": 1274436,
        "name": "Nassau Inter-County Express"
    },
    {"displayOnLoad": true, "id": 1274437, "name": "Bee-Line Bus System"},
    {"displayOnLoad": true, "id": 1274438, "name": "NJ Transit Bus"},
    {"displayOnLoad": true, "id": 1274441, "name": "New York City Bus"},
    {"displayOnLoad": true, "id": 1274439, "name": "New York City Bus"},
    {"displayOnLoad": true, "id": 1274440, "name": "AirTrain Newark"},
    {"displayOnLoad": true, "id": 1274442, "name": "AirTrain JFK"},
    {"displayOnLoad": true, "id": 1274443, "name": "AirTrain JFK"},
    {"displayOnLoad": true, "id": 1274444, "name": "NJ Transit Rail"},
    {"displayOnLoad": true, "id": 1274445, "name": "NJ Transit Rail"},
    {
        "displayOnLoad": true,
        "id": 1274446,
        "name": "Metro-North Railroad"
    },
    {
        "displayOnLoad": true,
        "id": 1274447,
        "name": "Metro-North Railroad"
    },
    {
        "displayOnLoad": true,
        "id": 1274449,
        "name": "Long Island Rail Road"
    },
    {
        "displayOnLoad": true,
        "id": 1274448,
        "name": "Long Island Rail Road"
    },
    {"displayOnLoad": true, "id": 1274450, "name": "PATH"},
    {"displayOnLoad": true, "id": 1274451, "name": "PATH"},
    {
        "displayOnLoad": true,
        "id": 1274452,
        "name": "Staten Island Railway"
    },
    {
        "displayOnLoad": true,
        "id": 1274453,
        "name": "Staten Island Railway"
    },
    {
        "displayOnLoad": true,
        "id": 1274454,
        "name": "New York City Subway"
    },
    {
        "displayOnLoad": true,
        "id": 1274455,
        "name": "New York City Subway"
    }
]
